curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/5ddb965c65d27e1512d44d9a.md

```diff
@@ -81,8 +95,25 @@ legend {
   flex-direction: column;
 }
 
-button:hover {
+button {
+  outline: none;
   cursor: pointer;
+  text-decoration: none;
+  background-color: var(--light-yellow);
+  border: 2px solid var(--dark-yellow);
+}
+
+.clear {
+  background-color: var(--light-pink);
+  color: var(--dark-red);
+  border-color: var(--dark-red);
+}
+
+button,
+input,
+select {
+  min-height: 24px;
+  color: var(--dark-text);
```

Comment:
If you do decided to up the variable name as suggested above then it will need to be updated here too